src/Text/Layout/Table/Cell/Formatted.hs

```diff
@@ -40,6 +43,29 @@ formatted
     -> Formatted a
 formatted = Format
 
+-- | Map over the formatting directives of a formatted value.
+reformat :: (String -> String)  -- ^ Function to operate on prefix text directives.
```

Comment:
This could be too unspecific (i.e. _what is reformatting?_).  This is essentially just mapping over the affixes.  How about `mapAffix` or `mapAffixes`?

Reply:
`mapAffixes` sounds reasonable to me. One use case is to remove all formatting directives, if rendering somewhere which does not support them. In hledger we have a couple of backends for rendering text.